RELEASING.md

```diff
@@ -12,29 +12,39 @@ Check [workflows/test-suite.yml](https://github.com/mongodb/terraform-provider-m
 ### Pre-release the provider 
 We pre-release the provider to make for testing purpose. **A Pre-release is not published to the Hashicorp Terraform Registry**.
 
-- Create and push the pre-release tag (`X.Y.Z-pre`) to master
+- Create and push the pre-release tag (`vX.Y.Z-pre`) to master
 ```bash
-git tag [YOUR_TAG]-pre
-git push origin [YOUR_TAG]-pre
+git tag vX.Y.Z-pre
+git push origin vX.Y.Z-pre
 ```
 
 - You will see the release in the [GitHub Release page](https://github.com/mongodb/terraform-provider-mongodbatlas/releases) once the [release action](.github/workflows/release.yml) has completed.
 
-### Generate the CHANGELOG.md 
+### Create PR updating Changelog and Upgrade Guide
+
+- Create a JIRA ticket and open a PR against the **master** branch. Make any manual adjustments if needed taking into account date format and format parameter names and resources/data source names if they begin with `mongodbatlas`.
+- Include the PM as a PR reviewer
+- Contact Documentation team in Slack to review the PR
+- Example: [#1478](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1478).
+
+The PR includes the following changes:
+
+#### Generate the CHANGELOG.md 
 We use a tool called [github changelog generator](https://github.com/github-changelog-generator/github-changelog-generator) to automatically update our changelog. It provides options for downloading a CLI or using a docker image with interactive mode to update the CHANGELOG.md file locally.
 
 - Update `since_tag` and `future-release` in [.github_changelog_generator](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github_changelog_generator)
 - **There is a bug with `github_changelog_generator` ([#971](https://github.com/github-changelog-generator/github-changelog-generator/issues/971))**: Make sure to update the `future-tag` with the pre-release tag. Once you generate the changelog, update `future-tag` with the final release tag in [.github_changelog_generator](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github_changelog_generator). Then, manually update the generated changelog to remove references to the pre-release tag
 - Run the following command: 
     ```bash 
-    docker run -it --rm -v "$(pwd)":/usr/local/src/your-app githubchangeloggenerator/github-changelog-generator -u mongodb -p terraform-provider-mongodbatlas -t <GH_TOKEN> --breaking-labels "breaking-change" --enhancement-label "**Enhancements**" --bugs-label "**Bug Fixes**"  --issues-label "**Closed Issues**" --pr-label "**Internal Improvements**"
+    docker run -it --rm -v "$(pwd)":/usr/local/src/your-app githubchangeloggenerator/github-changelog-generator -u mongodb -p terraform-provider-mongodbatlas -t <GH_TOKEN> --breaking-labels "breaking-change" --enhancement-label "**Enhancements**" --bugs-label "**Bug Fixes**"  --issues-label "**Closed Issues**" --pr-label "**Internal Improvements**" --max-issues 1000
```

Comment:
added a `--max-issues 1000` config as I was reaching an API rate limit (even when configuring a personal GH token). By default it fetches all (open and closed) issues in the repo (https://github.com/github-changelog-generator/github-changelog-generator/wiki/Advanced-change-log-generation-examples#additional-options)